sdcm/cql_stress_cassandra_stress_thread.py

```diff
@@ -123,7 +123,13 @@ def _run_cs_stress(self, loader, loader_idx, cpu_idx, keyspace_idx):  # pylint:
             loader.logdir, f'cql-stress-cassandra-stress-{stress_cmd_opt}-{log_id}.log')
 
         LOGGER.debug('cassandra-stress local log: %s', log_file_name)
-
+        remote_hdr_file_name = f"hdrh-cs-{stress_cmd_opt}-{log_id}.hdr"
```

Comment:
`cs` is expected to mean `cassandra-stress`, but here we use `cql-stress` tool.
Better to rename it.

Reply:
renamed `hdrh-cscs-...`